data_pipeline/twitter_ads_api/twitter_ads_api_config.py

```diff
@@ -9,8 +9,7 @@
 
 class TwitterAdsApiParameterValuesConfig(NamedTuple):
     from_bigquery: BigQuerySourceConfig = {}
-    start_date_value: Optional[str] = None
-    end_date_value: Optional[str] = None
+    ending_period_per_day: Optional[int] = 0
```

Comment:
naming:
  The suffix `in_days` might be better? (`per` suggests to me a breakdown or denominator)
  At the beginning `max` might be better? (a period has a start and an end, `ending period` may suggest to me there are multiple period and this refers to the the last period within a larger context, e.g. "ending period of 20th century")

Reply:
`maxPeriodInDays`